.github/workflows/publish-charm.yaml

```diff
@@ -30,26 +30,34 @@ jobs:
     steps:
       - name: Checkout
         uses: actions/checkout@v4
+        with:
+          ref: ${{ inputs.branch-name }}
+
       - name: Install charmcraft
         run: sudo snap install charmcraft --classic
+
       - name: Fetch Tested Charm
         uses: actions/download-artifact@v4
         with:
           name: tested-charm
+
       - name: Move charm in current directory
         run: find ./ -name ${{ inputs.charm-file-name }} -exec mv -t ./ {} \;
+
       - name: Sanitize the branch variable
         id: sanitize
         run: |
           echo sanitized=$(echo ${{ inputs.branch-name }} | sed 's/[/_.]/-/g' | sed "s/^/\//") >> $GITHUB_OUTPUT
-        if: ${{inputs.branch-name}} != ""
+        if: ${{ (inputs.branch-name != "") && (inputs.branch-name != "v1.4") }}
```

Comment:
Should we make the rule if the branch name starts with `v`?

Reply:
This branch is only used by v1.4 (and no other release will use it), so it doesn't matter. For the next branches, we should consider that however.

Reply:
Got it. Sounds good